guides/how-to-readme.md

```diff
@@ -0,0 +1,77 @@
+---
+explains: The goto file to see what a project is and how to use it
+---
+
+# How to write a `README`
+
+The `README` is the first thing almost anyone sees and perhaps one of the critical parts of the project. It serves as an advertisement for the product, indicator of for what users it can be relevant, the primary user’s guide and the way to get new collaborators up to speed.
+
+When writing the `README` keep your users in mind, they include:
+
+* Your teammates
+* Developers new to our organisation
+* (Potential) users of our projects
+* Contributors, both in and external
+* People interested in our work
+
+This guide goes into the primary elements any `README` should have, the bare minimum: what the project is; how to install, run and deploy; how to develop and test; and how to contribute.
+
+## 1.  The project name and description
+
+Give your project a short name by which people can refer to it. Preferably the repository name is similar. Descriptive names are great. However, feel free to get creative.
+
+The description is a high-level overview of what the project is for, what it does and in what way it should be used. The description can double as a mission to keep you focussed and on track, so it is wise to define this as early in the process as possible.
+
+> The `README` is an introduction. It should assume the reader knows absolutely nothing about the software and should provide a brief introduction. If software were a screenplay, the `README` would be the elevator pitch. If a person finishes reading the first 10 lines of `frobnicator/README` and still does not know if `frobnicator` is a widget library, accounting software, or a video game, then the author of the README has failed. — [William Pursell](https://stackoverflow.com/a/2305173)
+
+Be sure to answer these questions in your description:
+
+* What is this project?
+* How does it work?
+* Who are the users?
+* What is the benefit for the users?
+* What is the goal of this project?
+
+Start with the most important things so people can continue their search if this project is not what they are looking for.
+
+Make this section shine: If your project has a front-end include a screenshot here; if there is a demo or there are live examples, include links; if there are features, list them.
+
```

Comment:
This is the second best paragraph. When a project includes a demo it gets a lot stronger.

Basically developers get attracted by a short pitch, a good demo and an easy installation. If that seems OK, a developer will look at how active is the project. This is denoted by #contributors, branches, pull requests, age of pull requests and issues etcetera

Reply:
Agree, feel this is represented, however for information hierarchy I think this sentence fits best here.